Modules/rat_rescue.py

```diff
@@ -28,17 +28,18 @@ class Rescue(object):
     A unique rescue
     """
 
-    def __init__(self, case_id: UUID, client: str, system: str, irc_nickname: str,
-                 board: 'RatBoard' = None, created_at: datetime = None, updated_at: datetime = None,
-                 unidentified_rats=None, active=True, quotes: list = None, is_open=True, epic=False,
-                 code_red=False, successful=False, title: str = '',
-                 first_limpet: UUID or None = None, board_index: int = None,
-                 mark_for_deletion: dict or None = None, lang_id: str = "EN", rats: list = None):
+    def __init__(self, case_id: UUID, client: str, system: str, irc_nickname: str, board: 'RatBoard' = None,
+                 created_at: datetime = None, updated_at: datetime = None, unidentified_rats=None, active=True,
+                 quotes: list = None, epic=False, title: str = '', first_limpet: UUID or None = None,
+                 board_index: int = None, mark_for_deletion: dict or None = None, lang_id: str = "EN",
+                 rats: list = None, status: Status = Status.OPEN, code_red=False):
```

Comment:
The outcome should still be part of the constructor. Otherwise, commands handling already finished rescues can't be handled properly. Plus, if you hardcode it there's no point in us tracking it in the first place.

I personally also believe that there should be a setter for it, so that, in the future, a command changing the outcome of an already finished case is possible. (And I don't believe in limiting developers for any other reason than hiding implementation or providing compatability.) This point may require further discussion in #popcorn, though.

Reply:
if we need to expose this functionality, we can refactor it in later. As we discussed in IRC we are at a difference in methodology. Mecha has no reason to be modifying that property, and should always set it to None. If we need to change this functionality, we can refactor it in at a later point in time.